pkg/aws/ec2/mock/mock.go

```diff
@@ -335,3 +336,17 @@ func (e *API) GetSubnets() (types.SubnetMap, error) {
 	}
 	return subnets, nil
 }
+
+func (e *API) TagENI(eniID string, eniTags map[string]string) error {
+	e.rateLimit()
+	e.simulateDelay(TagENI)
+
+	e.mutex.Lock()
+	defer e.mutex.Unlock()
```

Comment:
`e.mutex.RUnlock()`

Reply:
@ungureanuvladvictor was this addressed?

Reply:
Just pushed the updated version.